python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -217,6 +220,14 @@ def _fsproxy_state(self) -> _State:
         elif v is not _DELETE:
             cls_dict[k] = v
 
+    for slow_name in dir(slow_type):
+        if slow_name in cls_dict or slow_name.startswith("__"):
```

Comment:
For arithmetic dunder methods like `__add__`, wouldn't we want to create a `_FastSlowAttribute` for that?

Reply:
So those are defined in the base class `_FastSlowProxy` now concretely (so it's already in the class dictionary).